rtv/content.py

```diff
@@ -298,21 +298,41 @@ def __init__(self, name, submissions, loader=default_loader):
 
     @classmethod
     def from_name(cls, reddit, name, loader=default_loader):
+        
+        display_type = 'normal'
 
         if name == 'front':
             return cls('Front Page', reddit.get_front_page(limit=None), loader)
 
         if name == 'all':
             sub = reddit.get_subreddit(name)
-
+        
         else:
+            
+            if '/' in name:
+                parse = name.split('/')
+                name = parse[0]
+
+                if parse[1] == 'top':
+                    display_type = 'top'
+   
+                elif parse[1] == 'new':
+                    display_type = 'new'
+
             try:
                 with loader():
                     sub = reddit.get_subreddit(name, fetch=True)
             except praw.errors.ClientException:
                 raise SubredditNameError(name)
+        
+        if display_type == 'top':
+            return cls('/r/'+sub.display_name, sub.get_top_from_all(limit=None), loader)
```

Comment:
I think it would be good to have the type in the display name to give the user some feedback e.g.
```
'/r/'+sub.display_name+'/top'
```